src/materialized/src/prof.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use lazy_static::lazy_static;
+
+use jemalloc_ctl::raw;
+use std::os::unix::ffi::OsStrExt;
+use std::sync::Arc;
+use std::sync::Mutex;
+use std::{ffi::CString, time::Instant};
+use tempfile::NamedTempFile;
```

Comment:
stylistically we normally break uses into "std", "external", "current crate", which would make this into two newline-delimited groups with `std` in one group and everything else in the other:

```suggestion
use std::os::unix::ffi::OsStrExt;
use std::sync::Arc;
use std::sync::Mutex;
use std::{ffi::CString, time::Instant};

use lazy_static::lazy_static;
use jemalloc_ctl::raw;
use tempfile::NamedTempFile;
```